changelogs/client_server/newsfragments/1746.feature

```diff
@@ -0,0 +1 @@
+Add support for recursion on the /relations endpoints (MSC3981).
```

Comment:
I've applied the change to `main` as part of release checklist steps, but for future PRs please use a consistent changelog style (and apologies if these aren't documented anywhere - we should fix that if undocumented).

* Use backticks around endpoints, and indicate the request method.
* Use `, as per [MSC3981](...).` at the end of your changelog, linking to the MSC in question.

```diff
Add support for recursion on the `GET /relations` endpoints, as per [MSC3981](https://github.com/matrix-org/matrix-spec-proposals/pull/3981).
```